index.html

```diff
@@ -48,5 +48,206 @@ <h5>I am a Software Developer</h5>
+            <p class="font-inter">html</p>
+          </div>
+          <div class="box">
+            <p class="font-inter">boostrap</p>
+          </div>
+          <div class="box">
+            <p class="font-inter">Ruby</p>
+          </div>
+        </div>
+        <button class="primary-button primary-color font-inter">See Project</button>
+      </div>
+    </div>
+    <div class="grid-container">
+      <div class="grid-items d-flex">
+        <div class="grid-item-card text-white">
+          <div class="grid-item-card-content">
+            <h2 class="font-crete-round">Profesional Art Printing Data</h2>
+            <p class="font-inter">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <div class="box-container d-flex">
+              <div class="box">
+                <p class="font-inter text-white">html</p>
+              </div>
+              <div class="box">
+                <p class="font-inter">boostrap</p>
+              </div>
+              <div class="box">
+                <p class="font-inter">Ruby</p>
+              </div>
+            </div>
+          </div>
+          <button class="primary-button primary-color font-inter">See Project</button>
+        </div>
+      </div>
+      <div class="grid-items d-flex">
+        <div class="grid-item-card text-white">
+          <div class="grid-item-card-content">
+            <h2 class="font-crete-round">Profesional Art Printing Data</h2>
+            <p class="font-inter">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <div class="box-container d-flex">
+              <div class="box">
+                <p class="font-inter text-white">html</p>
+              </div>
+              <div class="box">
+                <p class="font-inter">boostrap</p>
+              </div>
+              <div class="box">
+                <p class="font-inter">Ruby</p>
+              </div>
+            </div>
+          </div>
+          <button class="primary-button primary-color font-inter">See Project</button>
+        </div>
+      </div>
+      <div class="grid-items d-flex">
+        <div class="grid-item-card text-white">
+          <div class="grid-item-card-content">
+            <h2 class="font-crete-round">Profesional Art Printing Data</h2>
+            <p class="font-inter">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <div class="box-container d-flex">
+              <div class="box">
+                <p class="font-inter text-white">html</p>
+              </div>
+              <div class="box">
+                <p class="font-inter">boostrap</p>
+              </div>
+              <div class="box">
+                <p class="font-inter">Ruby</p>
+              </div>
```

Comment:
- The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

Reply:
Well done